src/Renci.SshNet/Common/HostKeyEventArgs.cs

```diff
@@ -58,6 +69,11 @@ public HostKeyEventArgs(KeyHostAlgorithm host)
             {
                 FingerPrint = md5.ComputeHash(host.Data);
             }
+
+            using (var sha256 = CryptoAbstraction.CreateSHA256())
+            {
+                FingerPrintSHA256 = Convert.ToBase64String(sha256.ComputeHash(host.Data)).Replace("=", "");
```

Comment:
Why here is `.Replace("=", "")`? Do other programs work like that?

Reply:
Yes, other software works like this as well.

OpenSSH does this.
For example `ssh-keyscan github.com | ssh-keygen -l -f -` produces 43 characters long Base64 outputs (without trailing `=`):
```
# github.com:22 SSH-2.0-babeld-51390fc7
# github.com:22 SSH-2.0-babeld-51390fc7
# github.com:22 SSH-2.0-babeld-51390fc7
# github.com:22 SSH-2.0-babeld-51390fc7
# github.com:22 SSH-2.0-babeld-51390fc7
3072 SHA256:uNiVztksCsDhcc0u9e8BujQXVUpKZIDTMczCvj3tD2s github.com (RSA)
256 SHA256:p2QAMXNIC1TJYWeIOttrVc98/R1BUFWu3/LiyKgUfQM github.com (ECDSA)
256 SHA256:+DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU github.com (ED25519)
```


PuTTY does this as well (displays 43 characters long Base64 output without trailing `=`) - when connecting to github.com this text is shown:
```
The host key is not cached for this server:

github.com (port 22)

You have no guarantee that the server is the computer you think it is.

The server's ssh-ed25519 key fingerprint is:

ssh-ed25519 255 SHA256:+DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU

If you trust this host, press "Accept" to add the key to PuTTY's cache and carry on connecting.

If you want to carry on connecting just once, without adding the key to the cache, press "Connect Once".

If you do not trust this host, press "Cancel" to abandon the connection.
```


WinSCP also does this - when connecting to github.com it displays these fingerprints:
```
ssh-ed25519 255 +DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU
ssh-ed25519 255 65:96:2d:fc:e8:d5:a9:11:64:0c:0f:ea:00:6e:5b:bd
```